<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeStatus</class>
 <widget class="QWidget" name="NodeStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="nodeListWidget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="nodeInfoWidget" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="structureGraphWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
